### PR TITLE
Create a query for listing rules assigned for a specific package

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/adviser/pull/1877

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -6270,3 +6270,31 @@ def get_python_rule_all(
                 result.append(self._rule_to_dict(rule))
 
             return result
+
+    def get_python_package_version_solver_rules(
```
Comment (pacospace) [inline thoth/storages/graph/postgres.py:6274]: suffix `_all()` should be added at the end based on convention: https://github.com/thoth-station/storages/tree/master/docs/conventions

**Review by pacospace — COMMENTED** [review]
lgtm!! 💯 one small comment

File: thoth/storages/graph/postgres.py (hunk 1 above)
Comment (fridex) [reply thoth/storages/graph/postgres.py]: Adjusted in 9e09060 PTAL 👍🏻

**Review by pacospace — APPROVED** [review]
Thanks!! :100

**Pull request merged**